packages/signers/test/ethereum.test.ts

```diff
@@ -0,0 +1,28 @@
+import test from "ava"
+import { ethers } from "ethers"
+import type { ActionPayload } from "@canvas-js/interfaces"
+import { EthereumActionSigner } from "@canvas-js/signers"
+import { verifyActionSignature } from "@canvas-js/verifiers"
+
+test("Sign an action for ethereum", async (t) => {
+	const sessionWallet = ethers.Wallet.createRandom()
+
+	const ethersWallet = ethers.Wallet.createRandom()
+	const wallet = new EthereumActionSigner(ethersWallet)
```

Comment:
it seems like the "sessionWallet" here is actually the main wallet, and "ethersWallet" is the session wallet, which is used to sign actions, and "wallet" is actually just a signer